partcad/src/partcad/converter.py

```
@@ -0,0 +1,169 @@
+
+        if file_format == "step":
+            return CADConverter._read_step(file_path)
+        elif file_format == "stl":
+            return CADConverter._read_stl(file_path)
+        elif file_format == "brep":
+            return CADConverter._read_brep(file_path)
+        else:
+            raise ValueError(f"Unsupported input format: {file_format}")
+
+    @staticmethod
+    def _write_shape(shape: TopoDS_Shape, file_path: str, file_format: str):
```

Ruff (0.8.2)

Missing return type annotation for staticmethod `_write_shape`

Add return type annotation: `None`

(ANN205)